SignalShareExtension/ShareViewController.swift

```diff
@@ -77,10 +76,7 @@ public class ShareViewController: UINavigationController, SAELoadViewDelegate, S
 
         // We don't need to use "screen protection" in the SAE.
 
-        contactsSyncing = OWSContactsSyncing(contactsManager:Environment.current().contactsManager,
-                                             identityManager:OWSIdentityManager.shared(),
-                                             messageSender:Environment.current().messageSender,
-                                             profileManager:OWSProfileManager.shared())
+        OWSContactsSyncing.sharedManager()
```

Comment:
nit: This reads like a no-op. But we're actually calling this getter here for it's side affects.  Can you add a comment, to lessen the chances that it get's inadvertently shuffled around.

Reply:
Done.